working_with_health_tool/main.bal

```diff
@@ -0,0 +1,49 @@
+import ballerina/http;
+import ballerinax/health.fhir.r4;
+import healthcare_samples/carinbb_package as carinbb;
+
+listener http:Listener httpListener = new (9090);
+
+public type Patient record {
+    string id;
+    string lastName?;
+    string firstName;
+    string middleName;
+    string gender?;
+};
+
+service / on httpListener {
+
+    isolated resource function get [string fhirType]/[string id]() returns @http:Payload {mediaType: ["application/fhir+json", "application/fhir+xml"]}r4:FHIRWireFormat|error {
```

Comment:
space before `r4:FHIRWireFormat` is missing

Reply:
fixed in [82be790](https://github.com/ballerina-guides/healthcare-samples/pull/15/commits/82be790415e7ca789008605060f62b95eded8c8a)